expr/expressions_test.go

```diff
@@ -60,7 +60,7 @@ func ExampleExpression_scalarFunction() {
 				"extensionFunction": {
 					"extensionUriReference": 1,
 					"functionAnchor": 2,
-					"name": "add"
+					"name": "add:i32_i32"
```

Comment:
Compound names are now required by the spec in the protobuf. Simple names should still be parsed in the protobuf, but in the round trip it will be turned back into a compound name, hence the update here